src/popup/router/pages/Notifications.vue

```diff
@@ -2,7 +2,7 @@
   <div class="popup">
     <ae-list class="noti-list">
       <ae-list-item fill="neutral" v-for="(noti, i) in notifications" :key="i" class="noti">
-        <img :src="icon" />
+        <img src="../../../icons/icon_48.png" />
```

Comment:
Same question goes here?

> 

Shouldn't this also be imported as the rest of the resources?